docs/common_issues.rst

```diff
@@ -88,6 +108,22 @@ default manager returns a filtered set), you can specify which manager to use wi
     >>> data = [PollWithAlternativeManager(id=x, question='Question ' + str(x), pub_date=now()) for x in range(1000)]
     >>> objs = bulk_create_with_history(data, PollWithAlternativeManager, batch_size=500, manager=PollWithAlternativeManager.all_polls)
 
+If your using `additional fields in historical models`_ and have one value to batch update
+into the history. Pass the a optional dict ``custom_historical_attrs`` with the key and value.
+A field ``ip_address`` would be passed as
+`custom_historical_attrs={'ip_address': '127.0.0.1'}`.
+
+.. _additional fields in historical models: historical_model.html#adding-additional-fields-to-historical-models
+
+.. code-block:: pycon
+
+    >>> bulk_update_with_history(
+            data, PollWithHistoricalSessionAttr,
+            custom_historical_attrs={'session': 'jam'}
+        )
+    >>> data[0].history.get().session
```

Comment:
The objects in `data` likely already have history records, so we should ideally use `latest()` in this example to not induce a `MultipleObjectsReturned` error (this is, of course, in contrast to using `bulk_create_with_history()` above).
```suggestion
    >>> data[0].history.latest().session
```